### ISSUE TITLE
Broken placeholders

### ISSUE THREAD

**Issue by MithrandirCraft** [GeorgH93/MarriageMaster#226]
Server version: Paper version git-Paper-232 (MC: 1.17.1)
MarriageMasterVersion: 2.4.2-Release

*Online mode*: no

*BungeeCord*: yes

https://github.com/kangarko/ChatControl-Red/issues/1325 

Ok, so basically I understand the ticket my staff made on CHC RED is terrible, so I'll proceed to explain myself:
The placeholders MarriageMaster offers don't work on ChatControl's formatting.

"Placeholder not found" is returned when inputting ingame /papi parse me %marriagemaster_Heart%

Affected placeholders are (as inputed in ChatControlRed, but you can imagine them with '%'):
{marriagemaster_Heart} {marriagemaster_IsMarried} 

I already reported this once in the past, and we fixed it, it seemed to be an issue with ChatControl: https://github.com/GeorgH93/MarriageMaster/issues/72

**Comment by GeorgH93** [comment]
This must come from one of your other placeholder providers.
If papi is unable to find the placeholder it will simply not replace it and marriage master does not have that message. Unless you have set the not married placeholder text in your language file to "Placeholder not found" this is not caused by marriage master.

**Comment by MithrandirCraft** [comment]
For future reference:
- The issue emerged right after updating MarriageMaster + Paper to 1.17.1 respective versions.
- The issue was solved on our side by deleting MarriageMaster's entire plugin folder. (basically reinstalling MarriageMaster and reconfiguring). It could not be solved by only resetting locale.yml (so maybe it was some issue with config.yml, even though we did no changes besides updating the plugin's .jar and server software).
